_data/sites.json

```diff
@@ -12817,6 +12817,15 @@
         ]
     },
 
+    {
+        "name": "wappalyzer",
+        "url": "https://www.wappalyzer.com/account/",
+        "difficulty": "easy",
+        "notes": "Click on your e-mail on the top right in the navigator. Find and click on Account. Scroll down to find the delete account button in red. Confirm deletion. Enter password to complete deletion.",
+        "domains": [
+            "wappalyzer.com"
+        ]
+    },
```

Comment:
```suggestion
    },

```

Reply:
Sadly I reverted the whitespaces, but also the line break and can't fix it on mobile. Can you add just a line break here without whitespaces, please, like the suggestion @mbtrs ?